src/RootCloseWrapper.js

```diff
@@ -15,7 +15,11 @@ function isModifiedEvent(event) {
   return !!(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey);
 }
 
-export default class RootCloseWrapper extends React.Component {
+/**
+ * The `<RootCloseWrapper/>` registers your callback on the document when rendered. Powers the
```

Comment:
possibly both – and also describe the purpose of root close wrapper (i.e. to make popovers behave in a modal manner where they close when the user tries to interact with the rest of the doucment)